src/axe-raw-sarif-converter.test.ts

```diff
@@ -127,10 +128,10 @@ describe('AxeRawSarifConverter', () => {
 
         it('outputs a sarif log whose run uses the axeToolPropertyProvider to populate the tool property', () => {
             const axeToolPropertyProviderMock: IMock<
-                () => Sarif.ToolComponent
+                (environmentData: EnvironmentData) => Sarif.ToolComponent
             > = Mock.ofInstance(getAxeToolProperties);
             axeToolPropertyProviderMock
-                .setup(ap => ap())
+                .setup(ap => ap(It.isAny()))
```

Comment:
I probably don't have enough knowledge about the code on this repo but I've always a little nervous about using `It.isAny()` (unless the verifiable is being called with `Times.never()`). 
In this case, I see there is a `stubEnvironmentData` being set on line 124 inside the `beforeEach` and then being pass to `testSubject.convert` in this test but base on the mock setup and the expect, I don't see that being asserted which, in addition to the newly added `It.isAny` make me wonder if I'm missing something here or maybe it's worth it change that param assertion for a stronger one or maybe we're lacking on more `expect` call in this test.